test/integration/bootstrap/bootstrap_test.go

```diff
@@ -100,6 +115,25 @@ func TestBootstrap(t *testing.T) {
 			}
 
 			bootstrap.DefaultApply(assert)
+
+			// configure options to push state to GCS bucket
+			tempOptions := bootstrap.GetTFOptions()
+			tempOptions.BackendConfig = map[string]interface{}{
+				"bucket": bootstrap.GetStringOutput("gcs_bucket_tfstate"),
+			}
+			tempOptions.MigrateState = true
+			// create backend file
+			cwd, err := os.Getwd()
+			require.NoError(t, err)
+			destFile := path.Join(cwd, "../../../0-bootstrap/backend.tf")
+			fExists, err2 := fileExists(destFile)
+			require.NoError(t, err2)
+			if !fExists {
+				srcFile := path.Join(cwd, "../../../0-bootstrap/backend.tf.example")
+				_, err3 := exec.Command("cp", srcFile, destFile).CombinedOutput()
```

Comment:
Do we need to exec copy? Why not just write a file if not exists with a backend config string (we can fix in a follow up after testing).

Reply:
we are using `cp` to also validate the file `backend.tf.example`

Reply:
@daniel-cit we should just read `backend.tf.example` and write to backend.tf. If any of that call path fails, we can fail the test.